README.md

```diff
@@ -2,4 +2,6 @@
 
 Specification can be found at the [`book`](https://github.com/0xPARC/pod2/tree/main/book) directory, a live rendered version at: https://0xparc.github.io/pod2/
 
-Requires Rust nightly: `rustup override set nightly`.
+## Usage
+- Requires Rust nightly: `rustup override set nightly`
```

Comment:
We can add a file like this https://github.com/openvm-org/openvm/blob/main/rust-toolchain.toml to set a particular rust nightly version and make jumping into the repo to develop more smooth.

Reply:
right, updated (removed the line from readme, added the `rust-toolchain.toml` file)